docs/openapi.yaml

```diff
@@ -865,14 +922,17 @@ components:
             properties:
               id:
                 type: string
+                format: uri
               name:
                 type: string
         description:
           type: string
         doi:
           type: string
-        id:
+          format: uri
+        identifier:
```

Comment:
I could not find a "id" on publication. It should be "identifier" - right?

Reply:
PublicationResponse implements WithId that adds the `id` field.